aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -144,14 +185,15 @@ def execute(self):
         """
         Runs the action.
 
-        :raises lambda_builders.actions.ActionFailedError: when .npmrc copying fails
+        :raises lambda_builders.actions.ActionFailedError: when copying fails
         """
 
         try:
-            npmrc_path = self.osutils.joinpath(self.source_dir, ".npmrc")
-            if self.osutils.file_exists(npmrc_path):
-                LOG.debug(".npmrc copying in: %s", self.artifacts_dir)
-                self.osutils.copy_file(npmrc_path, self.artifacts_dir)
+            for filename in [".npmrc", "package-lock.json", "npm-shrinkwrap.json"]:
```

Comment:
Can you do a quick check to see if `npm pack` packs an `npm-shrinkwrap.json`. If it does then we don't need it here.

Reply:
good call, it does include it. I guess that's why originally I didn't copy over this one 😂